apps/explorer/src/pages/AddressInfoPage/NFTDetailsModal.tsx

```diff
@@ -99,7 +99,7 @@ const NFTDetailsModal = ({ nft, collection, ...props }: NFTDetailsModalProps) =>
                 )}
               </>
             ) : (
-              <CardStyled>{t('Incorrect metadata')}</CardStyled>
+              <CardStyled>{t('Wrong / old format')}</CardStyled>
```

Comment:
Is it a french thing to add spaces around a `/`? I'd be in favor of "Wrong/old format"

Reply:
Why is this not needed in our other apps and only in the explorer?

Reply:
I think I remember a past discussion about filtering wrong or old metadata format in our wallets, but show them for debugging purposes in the explorer 🤔